licenses/constants.py

```diff
@@ -2,6 +2,6 @@
 # see i18n.utils, Locale.parse()
 EXCLUDED_LANGUAGE_IDENTIFIERS = ["nso", "oci-es", "x-i18n", "st"]
 # Exclude non 4.0 licenses for right now
-EXCLUDED_LICENSE_VERSIONS = ["3.0", "2.5", "2.1", "2.0", "1.0", ""]
+EXCLUDED_LICENSE_VERSIONS = ["3.0", "2.5", "2.1", "2.0", ""]
```

Comment:
I think we need to put back `1.0` and remove `""` from the list.

Reply:
Based on our conversation, it looks like `1.0` and `zero` are the same thing. Can we add a comment about this. in `cc-link-checker` we are using `zero` instead of `1.0`. It also seems the cc team has been referring to 1.0 licenses as zero licenses up until now.